html/global_attributes.json

```diff
@@ -81,7 +81,7 @@
               "version_added": null
             },
             "safari": {
-              "version_added": false
+              "version_added": "4"
```

Comment:
Do you test this?  My understand is Apple disable this by build flag.

Reply:
It appears to work in current iOS, and the information was taken from here as to initial support. https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/Attributes.html#//apple_ref/doc/uid/TP40008058-autocapitalize

Do you have better information? If so please point me to it and I'll take a look - thanks!

Reply:
> It appears to work in current iOS, and the information was taken from here as to initial support. https://developer.apple.com/library/archive/documentation/AppleApplications/Reference/SafariHTMLRef/Articles/Attributes.html#//apple_ref/doc/uid/TP40008058-autocapitalize
> 
> Do you have better information? If so please point me to it and I'll take a look - thanks!

This value is Safari for macOS, not iOS.

As long as I test this by wpt.live on Safari TP, macOS version isn't still supported. Also https://bugs.webkit.org/show_bug.cgi?id=180821 is for macOS support and source code is https://github.com/WebKit/webkit/blob/950143da027e80924b4bb86defa8a3f21fd3fb1e/Source/WTF/wtf/PlatformEnableCocoa.h#L86-L88, so it means macOS version is "not support". So you should keep "false" for macOS version.

Reply:
thanks! Updated.